### DISCUSSION TITLE
如何将这个部分隐藏？

### DISCUSSION THREAD

**Discussion by Rickustc** [ppoffice/hexo-theme-icarus#1291] [Q&A]
![image](https://github.com/ppoffice/hexo-theme-icarus/assets/68474394/c88d9bed-d835-40a3-b212-489849bb5c02)

**Comment by ppoffice** [comment]
注释掉这里：https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE/#%E6%96%87%E7%AB%A0%E8%AE%B8%E5%8F%AF%E5%8D%8F%E8%AE%AE
